### PR TITLE
Add integration test for Trec covid round 5

### PR DESCRIPTION
**Pull request by yuki617**

### CODE REVIEW

**Comment by lintool** [comment]
Confirmed that the following passes:

```
$ python -m unittest integrations/clprf/test_trec_covid_r5.py 
...
[topic][46] skipped
[topic][47] skipped
[topic][48] skipped
[topic][49] skipped
[topic][50] skipped
Copying over skipped topic 46 with 1000 lines
Copying over skipped topic 47 with 1000 lines
Copying over skipped topic 48 with 1000 lines
Copying over skipped topic 49 with 1000 lines
Copying over skipped topic 50 with 1000 lines
.
----------------------------------------------------------------------
Ran 1 test in 1855.202s

OK
```

File: integrations/clprf/test_trec_covid_r5.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
```
Comment (lintool) [inline integrations/clprf/test_trec_covid_r5.py:2]: I think we changed the tagline? https://github.com/castorini/pyserini/blob/master/integrations/clprf/test_trec_covid_r4.py#L2

File: integrations/clprf/test_trec_covid_r5.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                    -new_qrels {self.pyserini_root}/tools/topics-and-qrels/qrels.covid-round5.txt \
+                    -base {self.tmp}/runs/covidex.r5.d2q.1s \
+                    -tmp_base {tmp_folder_name} \
+                    -qrels {self.pyserini_root}/tools/topics-and-qrels/qrels.covid-round4-cumulative.txt \
+                    -index {prebuilt_index_path} \
+                    -tag covidex.r5.d2q.1s \
+                    -output {self.tmp}/output.json')
+        with open(f'{self.tmp}/output.json') as json_file:
+            data = json.load(json_file)
+            self.assertEqual("0.3859\\n'", data['map'])
```
Comment (lintool) [inline integrations/clprf/test_trec_covid_r5.py:84]: Why is there a trailing single quote?

**Review by lintool — CHANGES_REQUESTED** [review]

File: integrations/clprf/test_trec_covid_r5.py (hunk 2 above)
Comment (yuki617) [reply integrations/clprf/test_trec_covid_r5.py]: Yea, there shouldn't be a trailing single quote. I also fixed the problem in trec_covid round 3-5.

**Review by lintool — APPROVED** [review]

**Pull request merged**